src/main/java/org/fogbowcloud/saps/engine/core/archiver/storage/PermanentStorage.java

```diff
@@ -2,31 +2,29 @@
 
 import java.io.IOException;
 import java.util.List;
-import org.fogbowcloud.saps.engine.core.archiver.storage.exceptions.PermanentStorageException;
 import org.fogbowcloud.saps.engine.core.archiver.storage.exceptions.TaskNotFoundException;
-import org.fogbowcloud.saps.engine.core.model.SapsImage;
 
 public interface PermanentStorage {
 
 	/**
 	 * This function tries to archive a task trying each dirs in order
 	 * (inputdownloading -> preprocessing -> processing).
 	 *
-	 * @param task task to be archived
+	 * @param taskId The Task's unique identifier
```

Comment:
the documentation of this interface is outdated.

Reply:
Updated it.